site/app/libraries/database/DatabaseQueries.php

```diff
@@ -7338,6 +7338,39 @@ public function getEmailListWithIds() {
         return $this->course_db->rows();
     }
 
+    /**
+     * Gets a list of emails with user ids for all active particpants in Submitty
+     * array
+     * @return array<mixed>
+     */
+    public function getUserIdEmailExists(string $email, string $user_id): array {
```

Comment:
This function name doesn't really match what the function does.  Is the intention to return a boolean indicating whether the user exists?